src/QMCHamiltonians/MomentumEstimator.h

```diff
@@ -69,18 +69,12 @@ class MomentumEstimator: public QMCHamiltonianBase
   VectorSoaContainer<RealType,2> phases;
   ///phases of vPos
   std::vector<VectorSoaContainer<RealType,2> > phases_vPos;
+  ///dims of a grid for k points
+  int kgrid;
```

Comment:
Can we move this also to putSpecial if is not used anywhere else?

Reply:
It's used in the makeClone function, which I did not change. Therefore, if it is still desired to be there then we cannot move it.

Reply:
Remove also the line in makeClone.

Reply:
Yes, I don't think there is any reason why the clone would need to have that information, so it can be put in the putSpecial and removed from the makeClone.